packages/web/src/scss/components/FileUploader/_FileUploaderInput.scss

```
@@ -85,10 +86,54 @@ $_field-name: 'FileUploaderInput';
     display: initial;
 }
 
+:is(.FileUploaderInput--disabled, .FileUploaderInput.is-disabled)
+    :is(.FileUploaderInput__message, [data-element='validator_message']) {
+    @include form-fields-tools.message-disabled();
+}
+
+:is(.FileUploaderInput--disabled, .FileUploaderInput.is-disabled) :is(.FileUploaderInput__helperText) {
+    @include form-fields-tools.helper-text-disabled();
+}
+
+// 3.
+// stylelint-disable selector-max-compound-selectors, selector-max-specificity
+:is(.FileUploaderInput--disabled, .FileUploaderInput.is-disabled)
+    > .FileUploaderInput__input
+    ~ .FileUploaderInput__dropZone
+    > svg,
```

This could be shortened to:

```suggestion
:is(.FileUploaderInput--disabled, .FileUploaderInput.is-disabled)
    > .FileUploaderInput__dropZone
    > svg,
```

(I know, there is the comment no. 3, but) Is that because of specificity?